catalog/jest.config.js

```diff
@@ -21,10 +21,7 @@ module.exports = {
     '.*\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
       '<rootDir>/internals/mocks/image.js',
   },
-  setupFilesAfterEnv: [
-    '<rootDir>/internals/testing/test-bundler.js',
-    '@testing-library/react/cleanup-after-each',
```

Comment:
why have you removed that? is it failing?

Reply:
Yes, it's failing

And https://github.com/testing-library/react-testing-library/releases/tag/v9.0.0